### PR TITLE
Subs styles

### LINKED ISSUE #3703: [Spree upgrade][Subs] State label not centered vs v1
## Description
Small detail but the state label is not correctly centered, and I don't feel the angle down are either (they do in v1).

## Steps to Reproduce
1. Go to /admin/subscriptions
2. Look at the label for state and the angle in items & order

## Animated Gif/Screenshot
v2 :

v1:

## Severity
bug-s4: it's annoying, but you can use it

## Your Environment
* Version used: v2
* Browser name and version: Firefox 65
* Operating System and version (desktop or mobile): Ubuntu 18.10

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Closes #3703

Fixes some more minor visual issues with subscriptions and the `2-0-stable` branch in general.

#### What should we test?

Subs pages regressions are fixed. The labels in orders page have been fixed slightly (index and edit pages). Font sizes in buttons are back to how they were in v1. (See pics below)

### CODE REVIEW

**Comment by Matt-Yorkley** [comment]
Screenshot-2019-04-10-02-05:05
Screenshot-2019-04-10-02-05:14
Screenshot-2019-04-10-02-49:03
Screenshot-2019-04-10-03-28:50
Screenshot-2019-04-10-03-19:11

**Comment by sigmundpetersen** [comment]
Needs a rebase after #3718

File: app/assets/stylesheets/admin/components/states_decorator.scss [hunk 1]
```diff
@@ -1,7 +1,18 @@
 @import "../variables";
 
-.state {
-  @extend .state;
+#subscriptions .state {
```
Comment (luisramos0) [inline app/assets/stylesheets/admin/components/states_decorator.scss]: if this is specific to subscriptions than the file name should include subscriptions.
I dont understand what .state was being extended but I think now, without the extend, the _decorator part of the filename is also gone.
So, this file should be renamed to subscritpions_state.scss or something like that ;-)

File: app/assets/stylesheets/admin/openfoodnetwork.css.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 input[type="submit"], input[type="button"], button, .button {
   cursor: pointer;
-  padding: 6.5px 15px;
+  font-size: 85%;
```
Comment (luisramos0) [inline app/assets/stylesheets/admin/openfoodnetwork.css.scss]: this is changing all buttons in the backoffice, probably worth mentioning it in the test section of the PR.

Comment (Matt-Yorkley) [reply app/assets/stylesheets/admin/openfoodnetwork.css.scss]: It's partially reverting the last change I made to try to fix the visual regression on the buttons, but this time it's better. I compared the CSS with v1 and found what was missing. I mentioned it in the test section. :+1:

**Comment by RachL** [comment]
Way better ! :D

One example :

image

**Pull request merged**